ros-ocp/values.yaml

```diff
@@ -462,4 +462,35 @@ jwt_auth:
     # TLS configuration for Keycloak communication
     tls:
       insecureSkipVerify: false
-      # caCert: ""  # Path to CA certificate
\ No newline at end of file
+      # caCert: ""  # Path to CA certificate
+
+# UI with OAuth Proxy
+ui:
+  replicaCount: 1
+
+  oauthProxy:
+    image:
+      repository: quay.io/openshift/origin-oauth-proxy
+      pullPolicy: IfNotPresent
+      tag: "latest"
+    resources:
+      limits:
+        cpu: "100m"
+        memory: "128Mi"
+      requests:
+        cpu: "50m"
+        memory: "64Mi"
+
+  app:
+    image:
+      repository: openshift/hello-openshift
```

Comment:
what is this?

Reply:
A temporary image for the UI.
My current effort is to add auth to the UI. We The actual UI image is on another effort.

For your attention, this PR is still WIP

Reply:
Ah ok, got it. Thanks for the clarification 😄